panels/configspec.ini

```diff
@@ -82,6 +87,7 @@ custom_av = string(default="Custom A/V Setup")
 
 [[presentation]]
 qa = string(default="Mostly just Question and Answer")
+group_discussion = string(default="Group Discussion with Moderator")
```

Comment:
This was added to default config to accommodate a new help block that asks panelists to specify who will moderate a panel. I figured this was probably common enough to include in the default config.